docs/installation.rst

```diff
@@ -145,6 +145,17 @@ the old version using the same steps as shown above.
 From Source
 -----------
 
+Operating System
+~~~~~~~~~~~~~~~~
+
+* :ref:`debian-ubuntu`
+* :ref:`fedora`
+* :ref:`opensuse-tumbleweed-leap`
+* :ref:`macos`
+* :ref:`freebsd`
+* :ref:`windows-10-s-linux-subsystem`
+* :ref:`cygwin`
+
```

Comment:
ok, that's better, now it is at least at one place.

Reply:
(can be further refined, but I need at some point a merge to trigger the next revision)

Reply:
https://github.com/borgbackup/borg/pull/7423#issuecomment-1463964589 if you could check/add that linking, we could actually close that issue. 1 sentence / 1 link (if we do not have that yet) is enough.